x-pack/plugins/file_upload/server/get_time_field_range.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { IScopedClusterClient } from 'kibana/server';
+export async function getTimeFieldRange(
```

Comment:
Is getTimeFieldRange endpoint even needed? Why not just use already exposed ES search endpoint and have a client function that contains this logic?

Reply:
yes i agree, the original ML version of this function takes more arguments and so I think justifies having it's own endpoint. i've stripped most of these out for the file_upload version as they're not needed.
I'll move this search to the client side in a follow up PR.